cv/qc-cv.obo

```diff
@@ -32,9 +42,6 @@ name: has_column
 id: has_optional_column
 name: has_optional_column
 
-[Typedef]
-id: has_units
-name: has_units
 
 
 [Term]
```

Comment:
Line 61 (GitHub doesn't let me add a comment there directly) should be renamed to "PSI-MS CV Quality Control Vocabulary" for consistency with PSI-MS-CV and the relationships later on here.

Reply:
Line 68 and following: Instead of "OBI:0001442 ! q-value", a better (similar) example would be one of the q-value terms from the PSI-MS-CV. For example: MS:1002354 ! PSM-level q-value

Reply:
Line 81 and following: Remove comments as per PSI-MS-CV advice?